draft-ietf-httpbis-priority.md

```diff
@@ -427,7 +427,8 @@ where to send registration requests.
 
 The Priority HTTP header field carries priority parameters (see {{parameters}}).
 It can appear in requests and responses. It is an end-to-end signal of the
-request priority from the client or the response priority from the server.
+requested response
```

Comment:
The overall focus seems to be on the priority as being about the priority of the response, even though the header field can appear in the request.  But the origin is authoritative on the actual response priority, so it feels weird to say "response priority from the client".

Reply:
Overrall the draft is attempting to articulate the client's view of the urgency/incremental based on what the client knows locally. It's half the picture and the client cannot do anything about what actually happens. This is not like, for example content negotiation or anything. At the time the header is consumed (say by an intermediary) there may be no response so to speak of. 

The term "requested response" feels weird to me given this document and HTTP semantics more broadly

Reply:
My local notes (prior to conversion to PR) prefaced this remark with something like "would we be willing to consider" and a section reference to section 3 to explain some of the intended motivation.
I'll attempt to ponder if I want to make a different suggestion or just drop it, over the next couple days.

(And to be clear: the intended term was "requested response priority" but I put in an arbitrary line break to keep any CI that looks at line lengths from complaining.  I think your point stands, regardless.)

Reply:
Perhaps reword the sentence more broadly -- "It is an end-to-end signal of each endpoint's local view of the response priority."

Reply:
I like that @MikeBishop. Although I might tweak it a bit to something like

"It is an end-to-end signal that indicates the endpoint's view on how HTTP responses should be prioritized."

Reply:
That SGTM.

Reply:
I pushed a new commit with (a one-word tweak of) Lucas' version.  Thanks for the improved suggestion!